conbench/tests/api/test_runs.py

```diff
@@ -79,7 +79,11 @@ def test_get_run(self, client):
                 run,
                 candidate_baseline_runs={
                     "fork_point": DEFAULT_BRANCH_PLACEHOLDER,
-                    "latest_default": DEFAULT_BRANCH_PLACEHOLDER,
+                    "latest_default": {
+                        "baseline_run_id": run.id,
```

Comment:
Open question: as this stands, if the contender run is already on the latest commit on the default branch, the `latest_default` baseline run will be itself. Is that okay? Or should I exclude the contender run from the search?

Reply:
IMO we should exclude that from the list, but if that's a lot of work to do it's ok if it's there. It's definitely cleaner to not have that there + have someone click and wonder why something is being compared to itself

Reply:
Done! Will push in a second